docs/developer_guide/why_make_h2integrate.md

```diff
@@ -1,26 +1,26 @@
-# Welcome to GreenHEART
+# Welcome to H2Integrate
```

Comment:
This whole page probably needs a rework due to the new context of the name, but let's tackle that separately: https://github.com/NREL/GreenHEART/issues/107